### ISSUE TITLE
Does this need to be Self Hosted?

### ISSUE THREAD

**Issue by CEbbinghaus** [andrewlock/auto-assign-issues#1]
I added it to my Repo with the Config and everything. But unfortunately it doesnt Seem to work at all so im wondering if i did something wrong or this is just not running and i have to host my own version?

**Comment by andrewlock** [comment]
Hey, no, you shouldn't have to self-host it. You should just have to install the app from here: https://github.com/apps/auto-assign-issues and select which repos to install it in.

**Comment by CEbbinghaus** [comment]
i did that and even Provided a .yml file (which i later deleted). but so far nothing has worked. i tried removing and adding it back but it still wont assign me. Is it Limited to Public Repositories?

**Comment by CEbbinghaus** [comment]
> Is it Limited to Public Repositories?

I just tried it on a public repo and it worked. so this is not available for Private Repositories?

**Comment by Racso** [comment]
+1. Isn't working on private repos.

**Comment by andrewlock** [comment]
Yeah, looks like it doesn't work on private repositories 🙁 I can't find anything in the GitHub app docs about why not, just [this tweet](https://twitter.com/devhub_app/status/1129499699931697152) describing the problem - GitHub locked down access to the APIs!

Struggling to figure out what I need to do to fix it...

**Comment by mohammed-larbi-jebari** [comment]
Hi @andrewlock,

Yes I confirm that it only works with the public repository,you can see the "auto-assign" application it work perfectly with private repository.

https://github.com/apps/auto-assign
